rochu_gripper_node:
  ros__parameters:
    rochu:
      name: '1'
      ip : '192.168.1.200'
      port: 502
      #In kPa
      max_effort: 100
      min_effort: 0 
   
